luthier/config/src/tables.lua

```diff
@@ -136,16 +136,15 @@ M.physical = {
     Logical tables are defined in table M.logical. They map logical table
     names to their configuration:
 
-    * name - The name of the table
-        Defaults to the configuration's key
+    * key - The name of the logical table
```

Comment:
I see what you're trying to do here, but this is misleading. It implies there's a key named `key` on the
table that contains the logical table name. Besides, this is redundant with `They map logical table
names to their configuration:` above.

Reply:
This is inconsistent from what I did in some of the other lua files. It might be caused by a copy-paste from an older file. 
The idea is to write: 
```markdown
    Physical tables are defined in table M.physical that map physical table's
    name (should be the same as the dataset names defined in druid's config
    file) to a dictionary of physical table configuration:
    * The table is keyed on the name of the physical table
    * description - Brief documentation about the physical table.
        Defaults to the configuration's table key
    ...
```
I will do that instead. By the way, is the above description correct? It might be outdated in the sense that we do not need to have the physical table name match with the apiName in druid store, as we now have the `logicalToPhysicalColumnNames`.

Reply:
The physical table name still needs to correspond to Druid table. The `logicalToPhysicalColumnNames` only allows translating *column* names.

Reply:
I see! 
I will be using the `* The table is keyed on the name of the logical table` instead.